packages/adapter-squid/src/constants/ErrorMessages.ts

```diff
@@ -0,0 +1,7 @@
+export const ERR_GET_ROUTE_FAILURE =
+   "Failed to get route from Squid using the specified configuration";
+export const ERR_TX_RECEIPT_MISSING =
+   "Transaction receipt is missing or invalid";
+export const ERR_GET_ROUTE_UNDEFINED =
+   "Route data returned from squid is undefined";
+export const UNKNOWN_SQUID_ERROR = "Unknown Squid error occurred";
```

Comment:
minor: slightly less verbose (no repeat of `export const`)

```suggestion
export default const ErrorMessages = {
    ERR_GET_ROUTE_FAILURE: "Failed to get route from Squid using the specified configuration",
    ERR_TX_RECEIPT_MISSING: "Transaction receipt is missing or invalid",
    ERR_GET_ROUTE_UNDEFINED: "Route data returned from Squid is undefined",
    UNKNOWN_SQUID_ERROR: "Unknown Squid error occurred"
}
```